### DISCUSSION TITLE
Does datadog4s work with Java 17?

### DISCUSSION THREAD

**Discussion by marko-asplund** [datadog4s/datadog4s#747] [General]
I'm running into an issue (https://github.com/avast/datadog4s/issues/746) with datadog4s that prevents us from upgrading to Java 17. Is anyone running datadog4s successfully on Java 17?

**Comment by marko-asplund** [comment]
https://avast.github.io/datadog4s/userguide.html#note-on-java-compatibility
seems to provide a temporary workaround.

**Comment by Mann-tech13** [comment]
To resolve this issue, you need to determine which library or component is causing the error and update it to a version that is compatible with Java 17. The library or component might need to be recompiled with the latest version of Java
